src/NuGet.Core/NuGet.PackageManagement/BuildIntegratedPackageReference.cs

```diff
@@ -40,6 +41,27 @@ public BuildIntegratedPackageReference(LibraryDependency dependency, NuGetFramew
             Dependency = dependency;
         }
 
+        /// <summary>
```

Comment:
This BuiltIntegratedPackageReference.cs change comes from Martin's [PR](https://github.com/NuGet/NuGet.Client/pull/3539). He already had PR for reference type unit tests and I'm using some of his code only for reference type project unit test.